### DISCUSSION TITLE
ERROR Unrecognized GUID

### DISCUSSION THREAD

**Discussion by dchapNL** [Taxel/PlexTraktSync#126] [Q&A] [answered]
I'm trying to get this working (1st time) and despite trying almost everything I can think of or have read about I keep getting the same error as per the title; ERROR Unrecognized GUID.

I've tried to run main.py with my plex scanner\agent set to different options as a hail mary but nothing works, always the same error. Any advice would be seriously appreciated. I am fairly new to Plex / Trakt so it could be anything I guess.

Thank you, everyone.

**Comment by glensc** [comment, marked as answer]
@dchapNL I guess your problem is you're not using the legacy agent:
- https://github.com/Taxel/PlexTraktSync/issues/56

you can try the fix:
- https://github.com/Taxel/PlexTraktSync/pull/87#issuecomment-797088395

**Reply by dchapNL** [reply]
@glensc  using Movies > Manage Library > Edit ....

https://prnt.sc/10jwlay

The screenshot above is my latest/current setup. I attempted main.py after with those settings and I still get the same error.

Also thank you very much for taking the time to reply, much appreciated.

**Reply by glensc** [reply]
@dchapNL if the fix did not help, please post the full error text here you are having.

**Reply by dchapNL** [reply]
@glensc Plex wasn't refreshing the metadata even though I requested a refresh multiple times, so everything was still using the old scan.  I realized this as a couple of titles that weren't using TMDB weren't giving the GUID error.

I'm in the process of rebuilding my library so hopefully, I can get the script to work following that. Thanks again for your help.

**Reply by dchapNL** [reply]
Followup:  I rebuilt my library adding the movies using the Legacy agent, once the metadata had downloaded, I ran the script and it worked perfectly.

**Reply by glensc** [reply]
Meanwhile, https://github.com/Taxel/PlexTraktSync/pull/87 got merged, so the other agent should work as well.

I'm personally staying with Legacy Agent, but other users have reported success.
